main.js

```diff
@@ -11,59 +13,46 @@ let orderListTodo = []
 
 makeOrderButton.addEventListener('click', makeNewOrderBox)
 
-function makeNewOrderBox(event) {
+function makeNewOrderBox() {
     if (makeOrder.value === '') {
         return
     }
 
     const newOrder = createOrder(makeOrder.value)
+
     main.appendChild(newOrder)
 
     makeOrder.value = ''
-
-    console.log(orderList)
 }
 
 function createOrder(text) {
-    const order = document.createElement('div')
-    const newId = Date.now().toString()
-    order.setAttribute('todo-id', newId)
-    order.setAttribute('is-done', 'false')
-    order.classList.add('box')
-
-    const orderBox = document.createElement('div')
-    orderBox.classList.add('box__text')
-    order.appendChild(orderBox)
-
-    const orderText = document.createElement('p')
-    orderText.classList.add('box__div')
-    orderBox.appendChild(orderText)
-    orderText.innerHTML = text
+    const newOrder = new Order(text)
+    const order = newOrder.render()
+    const newId = newOrder.getId()
+    const orderText = newOrder.getOrderText()
+    const trashButton = newOrder.getTrashButton()
+    const editButton = newOrder.getEditButton()
+    const editInput = newOrder.getEditInput()
 
     orderText.addEventListener('click', event => {
+
```

Comment:
you have access to all of the components via getters and still, you are using `event.target`